docs/discover/get-started-discover.asciidoc

```diff
@@ -0,0 +1,354 @@
+hover your mouse over the *Field* or *Value* columns for filters and additional options. 
+** Use the search above the table to filter for specific fields or values, or filter by field type using the options to the right of the search field.
+** You can pin some fields by clicking the left column to keep them displayed even if you filter the table.
++
+TIP: You can restrict the fields listed in the detailed view to just the fields that you explicitly added to the **Discover** table, using the **Selected only** toggle. In ES|QL mode, you also have an option to hide fields with null values.
+
+. To create a view of the document that you can bookmark and share, select ** View single document**.
+
+. To view documents that occurred before or after the event you are looking at, select
+**View surrounding documents**.
+
+
+
+
+[float]
+[[search-in-discover]]
+=== Search and filter data
+
+[float]
+==== Default mode: Search and filter using KQL
+
+One of the unique capabilities of **Discover** is the ability to combine
+free text search with filtering based on structured data.
+To search all fields, enter a simple string in the query bar.
+
+[role="screenshot"]
+image:images/discover-search-field.png[Search field in Discover]
+
+To search particular fields and
+build more complex queries, use the <<kuery-query,Kibana Query language>>.
+As you type, KQL prompts you with the fields you can search and the operators
+you can use to build a structured query.
+
+For example, search the ecommerce sample data for documents where the country matches US:
+
+. Enter `g`, and then select *geoip.country_iso_code*.
+. Select *:* for equals some value and *US*, and then click the refresh button or press the Enter key.
+. For a more complex search, try:
++
+```ts
+geoip.country_iso_code : US and products.taxless_price >= 75
+```
+
+[[filter-in-discover]]
+Whereas the query defines the set of documents you are interested in,
+filters enable you to zero in on subsets of those documents.
+You can filter results to include or exclude specific fields, filter for a value in a range,
+and more.
+
+For example, exclude results from the ecommerce sample data view where day of week is not Wednesday:
+
+. Click image:images/add-icon.png[Add icon] next to the query bar.
+. In the *Add filter* pop-up, set the field to *day_of_week*, the operator to *is not*,
+and the value to *Wednesday*.
++
+[role="screenshot"]
+image:images/discover-add-filter.png[Add filter dialog in Discover]
+
+. Click **Add filter**.
+. Continue your exploration by adding more filters.
+. To remove a filter, click the close icon (x) next to its name in the filter bar.
+
+[float]
+==== Search and filter using ES|QL
+
+You can use **Discover** with the Elasticsearch Query Language, ES|QL. When using ES|QL,
+you don't have to select a data view. It's your query that determines the data to explore and display in Discover.
+
+You can switch to the ES|QL mode of Discover from the application top bar.
```

Comment:
```suggestion
You can switch to the ES|QL mode of Discover from the application menu bar.
```
I know internally we've been referring to this as the "app menu", but not sure if we have a standardized term for the docs. Do you know @ryankeairns?

Reply:
I think it works like that, I'll check for consistency with other apps/docs